package/yast2-iscsi-client.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Thu Jun  4 14:34:47 UTC 2015 - lszhu@suse.com
+
+- Incoming authentication and outgoing authentication were set backwards (bnc#933576)
```

Comment:
Just a minor thing (compared to the already mentioned issue). Try to wrap this line in the same column that the ```----``` line.